frontend/packages/core/src/Assets/icons/ExperimentIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+
+import StyledSvg from "./helpers";
+
+const ExperimentIcon = (props: React.SVGProps<SVGSVGElement>) => (
```

Comment:
We should probably use the `SVGProps` from within Clutch since it will allow you to remove the height and width and rely on the `size` prop instead with predetermined height and widths